app/ports/templates/ports/includes/non-ajax-pagination.html

```diff
@@ -0,0 +1,69 @@
+<!-- ############
+This pagination is used on the following pages:
+#1. Category
+#2. Variant
+#3. Maintainer
+-->
+
+<div class="row">
+    <div class="col-lg-7">
+        <p>Page {{ objects.number }} of {{ objects.paginator.num_pages }} | Showing
+            objects {{ objects.start_index }} to
+            {{ objects.end_index }} of {{ all_count }}</p>
+    </div>
+    <div class="col-lg-5">
+        <form class="form-inline justify-content-lg-end" method="get"
+              action="">
+            <label for="paginate_by">Ports per page: </label>
+            <input style="max-width: 150px" class="ml-2" type="number" name="items"
+                   value="{{ paginate_by }}">
+            <button type="submit">Update</button>
+        </form>
+    </div>
+</div>
+
+
+
+{% if objects.has_other_pages %}
+    <nav aria-label="Page navigation">
+        <ul class="pagination">
+            {% if objects.has_previous %}
+                <li class="page-item"><a class="page-link"
+                                         href="?page={{ objects.previous_page_number }}&paginate_by={{ paginate_by }}">&laquo;</a>
```

Comment:
You said you changed the name, but `paginate_by` is still there in various places, and on http://ec2-52-34-234-111.us-west-2.compute.amazonaws.com/statistics/ports/ it's not possible to reference page or the number of items on the page.